frontend/src/navbar/navbar.js

```diff
@@ -66,6 +66,15 @@ module.exports = app => app.component('navbar', {
     },
     defaultRoute() {
       return this.allowedRoutes[0]?.name || 'dashboards';
+    },
+    hasTasks() {
+      // fix this when done
+      try {
+        require.resolve('@mongoosejs/task');
```

Comment:
This check needs to happen on the server side, frontend doesn't know about @mongoosejs/task.